react-client/.env.example

```
@@ -1,6 +1,2 @@
-# Base URL بک‌اند. اگه خالی باشه axios mutator میره روی '/api' و با پروکسی وایت تو dev کار می‌کنه.
-# برای بک‌اند لوکال روی پورت 3000 یا پروکسی وایت رو ست کن یا خط زیر رو از کامنت دربیار.
-# VITE_API_BASE_URL=http://localhost:3000
-VITE_API_BASE_URL=
-VITE_API_WITH_CREDENTIALS=false
-
+VITE_API_BASE_URL=http://localhost:3000
+VITE_API_WITH_CREDENTIALS=false
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)